extensions/amp-story/1.0/amp-story.js

```diff
@@ -2264,6 +2266,21 @@ export class AmpStory extends AMP.BaseElement {
       });
   }
 
+  /**
+   * Loads amp-story-education if the viewer capability is provided.
+   * @private
+   */
+  maybeLoadStoryEducation_() {
+    if (!this.viewer_.hasCapability('education')) {
+      return;
+    }
+
+    Services.extensionsFor(this.getAmpDoc().win).installExtensionForDoc(
```

Comment:
I'm not entirely sure this actually works for all cases, I'm currently investigating a bug where the `viewer-integration.js` extension in the open source player isn't loaded in canonical sites. I think it might have to do with the `this.getAmpDoc().win` bit. I think we should replace it with this `this.win`, but I'm still in the process of verifying.

Reply:
Cool, thanks. It looks like all other extensions use `this.win`, I replaced both.